cocotb/regression.py

```diff
@@ -180,34 +185,41 @@ def _discover_tests() -> Iterable[Test]:
                 _logger.info(traceback.format_exc())
                 raise
 
-            if test_str:
-
+            if tests:
```

Comment:
```suggestion
            if tests is not None:
```

Reply:
Changed in ccc0ce20dd809c137e93ae20666fd06190fa0f4e.